src/Params/ADnoteParameters.cpp

```diff
@@ -200,8 +200,8 @@ static const Ports voicePorts = {
             "Modulator Frequency Envelope"),
     rToggle(PFMAmpEnvelopeEnabled,   rShort("enable"), rDefault(false),
             "Modulator Amplitude Envelope"),
-
-
+    rToggle(PsyncEnabled,               rShort("sync"),  rDefault(false),
```

Comment:
Can you please rebase the PR on newest master? This will activate some more CI checks :smile:

Reply:
done